modules/imgcodecs/test/test_exr.impl.hpp

```diff
@@ -314,4 +314,27 @@ TEST(Imgcodecs_EXR, read_RGBA_unchanged)
     EXPECT_EQ(0, remove(filenameOutput.c_str()));
 }
 
+// See https://github.com/opencv/opencv/pull/26211
+// ( related with https://github.com/opencv/opencv/issues/26207 )
+TEST(Imgcodecs_EXR, imencode_regression_26207_extra)
+{
+    // CV_8U is not supported depth for EXR Encoder.
+    cv::Mat src(100, 100, CV_8UC1);
```

Comment:
I propose  to initialize mat with some values: some color or random data. It's applicable for the all new tests.
1. It makes test really deterministic
2. It makes sanitizers happy.

Reply:
Thank you for your comment, and I agree with you !

I fixed it, and set `const` for their source images.